src/wp-includes/functions.wp-scripts.php

```diff
@@ -166,20 +168,22 @@ function wp_add_inline_script( $handle, $data, $position = 'after' ) {
  *                                    as a query string for cache busting purposes. If version is set to false, a version
  *                                    number is automatically added equal to current installed WordPress version.
  *                                    If set to null, no version is added.
- * @param bool             $in_footer Optional. Whether to enqueue the script before `</body>` instead of in the `<head>`.
- *                                    Default 'false'.
+ * @param array             $args     {
+ *      Optional. An array of additional script loading strategies. Default empty array.
+ *
+ *      @type string    $strategy     Optional. Values blocking|defer|async. Default 'blocking'.
+ * }
  * @return bool Whether the script has been registered. True on success, false on failure.
  */
-function wp_register_script( $handle, $src, $deps = array(), $ver = false, $in_footer = false ) {
+function wp_register_script( $handle, $src, $deps = array(), $ver = false, $args = array() ) {
```

Comment:
https://github.com/WordPress/wordpress-develop/pull/4391/files#r1180443438

Reply:
Resolving as per @felixarntz reply to the linked suggestion above.